### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by surajTechlift**

### CODE REVIEW

File: erpnext_oralcare/erpnext_oralcare/doctype/conservative_dent_and_endo/conservative_dent_and_endo.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Characteristics"
+  }
+ ],
+ "istable": 1,
+ "modified": "2019-05-12 15:33:16.088184",
+ "modified_by": "Administrator",
+ "module": "Erpnext Oralcare",
+ "name": "CONSERVATIVE DENT AND ENDO",
```
Comment (palashjhabak) [inline erpnext_oralcare/erpnext_oralcare/doctype/conservative_dent_and_endo/conservative_dent_and_endo.json]: Make the doctype name proper

File: erpnext_oralcare/erpnext_oralcare/doctype/family/family.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+ "creation": "2019-05-15 01:11:55.274550",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "name1"
+ ],
+ "fields": [
+  {
+   "fieldname": "name1",
```
Comment (palashjhabak) [inline erpnext_oralcare/erpnext_oralcare/doctype/family/family.json]: name is reserved for id of the doctype so when you use "name" as the id of any field it changes to name1, name2 and so on. Dont use name ever change it to something more meaningful like family_name

File: erpnext_oralcare/erpnext_oralcare/doctype/new_doctype/new_doctype.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+   "fieldtype": "Data",
+   "label": "Opposing Teet",
+   "length": 100
+  }
+ ],
+ "is_submittable": 1,
+ "modified": "2019-05-13 13:10:51.263228",
+ "modified_by": "Administrator",
+ "module": "Erpnext Oralcare",
+ "name": "new Doctype",
```
Comment (palashjhabak) [inline erpnext_oralcare/erpnext_oralcare/doctype/new_doctype/new_doctype.json]: what is "new doctype"? If this is not meant to be checked in dont check in. If this is needed name it properly

File: erpnext_oralcare/erpnext_oralcare/doctype/conservative_dent_and_endo/conservative_dent_and_endo.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{
```
Comment (palashjhabak) [inline erpnext_oralcare/erpnext_oralcare/doctype/conservative_dent_and_endo/conservative_dent_and_endo.json]: Check for mandatory fields in all these new doctypes.

File: erpnext_oralcare/erpnext_oralcare/doctype/conservative_dent_and_endo/conservative_dent_and_endo.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019, Techlift and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+# import frappe
+from frappe.model.document import Document
+
+class CONSERVATIVEDENTANDENDO(Document):
```
[the remark this answers is not in the document]
Comment (surajTechlift) [reply erpnext_oralcare/erpnext_oralcare/doctype/conservative_dent_and_endo/conservative_dent_and_endo.py]: done

File: erpnext_oralcare/erpnext_oralcare/doctype/family/family.json (hunk 2 above)
Comment (surajTechlift) [reply erpnext_oralcare/erpnext_oralcare/doctype/family/family.json]: done

**Comment by surajTechlift** [comment]
updated above changes

**Pull request closed without merging**